cmd/scCallsExecutor/CLI.md

```diff
@@ -0,0 +1,37 @@
+
+# Bridge SC calls Executor CLI
+
+The **MultiversX Bridge SC calls executor** exposes the following Command Line Interface:
+
+```
+$ scCallsExecutor --help
+
+NAME:
+   SC calls executor CLI app - This is the entry point for the module that periodically tries to execute SC calls
+
+USAGE:
+   scCallsExecutor [global options] command [command options] [arguments...]
+
+VERSION:
+   undefined/go1.20.7/linux-amd64/03d1f4fa88
+
+AUTHOR:
+   The MultiversX Team <contact@multiversx.com>
+
+COMMANDS:
+   help, h  Shows a list of commands or help for one command
+
+GLOBAL OPTIONS:
+   --working-directory directory          This flag specifies the directory where the node will store databases and logs.
+   --log-level level(s)                   This flag specifies the logger level(s). It can contain multiple comma-separated value. For example, if set to *:INFO the logs for all packages will have the INFO level. However, if set to *:INFO,api:DEBUG the logs for all packages will have the INFO level, excepting the api package which will receive a DEBUG log level. (default: "*:DEBUG")
```

Comment:
```suggestion
   --log-level level(s)                   This flag specifies the logger level(s). It can contain multiple comma-separated values. For example, if set to *:INFO the logs for all packages will have the INFO level. However, if set to *:INFO,api:DEBUG the logs for all packages will have the INFO level, excepting the api package which will receive a DEBUG log level. (default: "*:DEBUG")
```

Reply:
changed